django_mongodb/base.py

```diff
@@ -72,6 +72,16 @@ class DatabaseWrapper(BaseDatabaseWrapper):
         "istartswith": "LIKE UPPER(%s)",
         "iendswith": "LIKE UPPER(%s)",
     }
+    # As with `operators`, these patterns are used to generate SQL before MQL.
+    pattern_esc = "%%"
```

Comment:
I can't find where this pattern is used.  Is it necessary ?

Reply:
https://github.com/django/django/blob/89557d4c66b469616fc3a16ba11b2999233efa8d/django/db/models/lookups.py#L577
(see comment on line 75)